examples/apps/mouse-drawing/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "mouse-drawing"
+publish = false
+license.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+color-eyre.workspace = true
+crossterm.workspace = true
+line_drawing = "1.0.0"
```

It would be nice to add a comment explaining what this crate does.

```suggestion
line_drawing = "1.0.0" # A collection of line-drawing algorithms
```